src/tools/nnfusion/distributed_training/mnist/README.md

```diff
@@ -0,0 +1,30 @@
+## Distributed MNIST Training Example
+### Install Dependencies
+```sh
+# install superscaler 
+$ git clone msrasrg@vs-ssh.visualstudio.com:v3/msrasrg/SuperScaler/SuperScaler && cd SuperScaler  && pip install . && cd - && rm -fr SuperScaler
+# install other dependencies
+$ pip install torch torchvision mpi4py
+```
+
+### Prepare data
+Prepare your own trainable frozen model or you can acquire from [mnist_mlp.onnx](https://to-be-replaced-here) and GPU cluster's specification file which describes the underlying topology of your model training environment.   Since the GPU cluster's specification is used by [SuperScaler](https://github.com/microsoft/SuperScaler.git), you can learn all the details from there. Or an example  [resource_pool.yaml](https://github.com/microsoft/SuperScaler#appendix-a-sample-resource_poolyaml) is provided.
```

Comment:
```suggestion
Prepare your own trainable frozen model or you can acquire from [mnist_mlp.onnx](https://nnfusion.blob.core.windows.net/models/onnx/mnist_mlp.onnx) and GPU cluster's specification file which describes the underlying topology of your model training environment.   Since the GPU cluster's specification is used by [SuperScaler](https://github.com/microsoft/SuperScaler.git), you can learn all the details from there. Or an example  [resource_pool.yaml](https://github.com/microsoft/SuperScaler#appendix-a-sample-resource_poolyaml) is provided.
```